spec/unit/resources/enclosure/import_certificate_spec.rb

```diff
@@ -1,20 +1,22 @@
 require_relative './../../../spec_helper'
-require 'FileUtils'
 
-describe 'oneview_test_api600_synergy::enclosure_import_ertificate' do
+describe 'oneview_test_api600_synergy::enclosure_import_certificate' do
   let(:resource_name) { 'enclosure' }
   let(:base_sdk) { OneviewSDK::API600::C7000 }
   include_context 'chef context'
+  let(:bay_number) { 1 }
   let(:csr_data) do
     {
       'type' => 'CertificateDataV2',
-      'base64Data' => 'encrypted_data'
+      'base64Data' => nil
     }
   end
 
   it 'imports a signed certificate' do
+    allow(File).to receive(:open).and_call_original
+    allow(File).to receive(:open).with('csr_data_file.txt').and_return(true)
```

Comment:
Not sure if we are using this file content in importing the certificate below. Can you please check to read and parse the file contents